I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 415, in script
    "1231$$$$$$$$!@#!@$!@%!@@@@@@@@@@@@@@@@@@%!!!!!!!!!!!!!!!!!!!!!!!!!!!!!%!%####################################!!!!!!!!!!!!!@####################"
ValueError: unsupported format character '!' (0x21) at index 21

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 415, in script
    "1231$$$$$$$$!@#!@$!@%!@@@@@@@@@@@@@@@@@@%!!!!!!!!!!!!!!!!!!!!!!!!!!!!!%!%####################################!!!!!!!!!!!!!@####################"
  File "C:\khp\renpy-8.1.0-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\khp\renpy-8.1.0-sdk\renpy\exports.py", line 1462, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character '!' (0x21) at index 21

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.0.23051307
kursach 1.0
Mon Jun  5 15:55:28 2023
